Apply 9800 default port, introduce `hosts` config for `logstash-output`. (#13)
logstash-plugins/logstash-integration-logstash: docs/output-logstash.asciidoc

&nbsp;

[id="plugins-{type}s-{plugin}-hosts"]
===== `hosts`

* Value type is <<string,string>>
* There is no default value for this setting.
* Constraints:

> ** When IPv6 is applied, IP address must be in an enclosed bracket.

Reviewer: ```suggestion
** When using IPv6, IP address must be in an enclosed in brackets.
```

> ** When any URL omits a port component, the default `9800` is used.

Reviewer: ```suggestion
** When a port is not provided, the default `9800` is used.
```

A downstream input {ls} host or IP address to connect.

> NOTE: Currently, plugin supports a single host. Multi-host support is coming soon.

Author: ```suggestion
NOTE: Single host endpoint is supported for `hosts`. Multi-host support is coming soon.
```

Host can be any of IPv4, IPv6 (in enclosed bracket) or host name, examples:

>       `"127.0.0.1"
>       `"127.0.0.1:9800"`
>       `"foo-bar.com"`
>       `"foo-bar.com:9800"`
>       `"[::1]"`
>       `"[::1]:9000"`
>       `"[2001:0db8:85a3:0000:0000:8a2e:0370:7334]"`

Reviewer 2: 1. An asciidoc list renders better than the block
2. use `*.example.com` instead of `foo-bar.com`
3. missing variant with bracketed uncompressed ipv6 with port

```suggestion
* `"127.0.0.1"`
* `"127.0.0.1:9801"`
* `"ds.example.com"`
* `"ds.example:9802"`
* `"[::1]"`
* `"[::1]:9803"`
* `"[2001:0db8:85a3:0000:0000:8a2e:0370:7334]"`
* `"[2001:0db8:85a3:0000:0000:8a2e:0370:7334]:9804"`
```
